### DISCUSSION TITLE
Spannungseinbruch bei Verschattung

### DISCUSSION THREAD

**Discussion by spcqike** [hoylabs/OpenDTU-OnBattery#888] [Show and tell]
Mahlzeit,

ich meine mich zu Erinnern, dass wir letztens irgendwo das Thema "Parallelschalten zweier Strings an einem Laderegler" hatten. Dabei wurde in den Raum geworfen, dass das nicht gut sei, da Teilverschattung das Ganze umwirft und nicht richtig geht. 
Die Aufzeichnung der Spannungsverläufe meiner kleinen Anlage (HM-600), die zeigt, dass die Spannung vom MPPT relativ konstant hoch gehalten wird, wurde mit "das ist ja keine richtige Verschattung, bei größeren Anlagen schwanken die Spannungen extrem" kommentiert.

Ich wollte dort Daten einer größeren Anlage (in Summe 300kVA, 430kWP) nachreichen, finde den Beitrag aber nicht mehr :D 

daher meine Erkenntnis und die Aufzeichnung hier :) 

![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/50098985/5830563f-27e1-4c9a-be63-bd3ac37be086)

gefiltert nach einem 125kVA Wechselrichter. oben seine Ausgangsleistung, unten die Spannung der ersten 5 MPPT (jeweils 2 Strings parallel, in Summe hat er 12 MPPT bzw. 24 Eingänge

wie man sieht, schwankt die MPPT Spannung bei Verschattung auch dort nur um ~20-30V, also bis zu 4%. obwohl die Leistung um 80% einbricht.

runter skaliert auf kleinere Anlagen denke ich daher immer noch, dass man mehrere Strings an einem größeren Laderegler parallel schalten kann, ohne dass es zu größeren Problemen kommt. der MPPT hält die Spannung, die nicht so stark einbricht oder steigt wie angenommen, und die verschatteten Module liefern einfach weniger Strom

**Comment by Manos1966** [comment]
Hi @spcqike,
meinst du das? 
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Victron-MPPT-battery-charger#eastwest-orientation-of-panels-with-only-one-victron-mppt-charger-

**Reply by spcqike** [reply]
Nein nicht im Wiki. War ne Diskussion irgendwo. 🤷‍♂️

**Reply by Manos1966** [reply]
🙄 😇 
![Capture](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/b7269d6c-d682-45bf-95f5-4a2d21679291)
